modules/fun.js

```diff
@@ -0,0 +1,450 @@
+    if (msg.content.toLowerCase().includes(`bananas`)) {
+      if (roll < 0.1) {
+        msg.channel.send({ files: [new Discord.AttachmentBuilder('media/buttermelonsMan.jpeg')] }).catch(u.errorHandler);
+      } else {
+        msg.channel.send(`*buttermelons`).catch(u.errorHandler);
+      }
+    } else if (msg.content.toLowerCase().includes(`banana`)) {
+      if (roll < 0.06) {
+        msg.channel.send({ files: [new Discord.AttachmentBuilder('media/buttermelonPile.png')] }).catch(u.errorHandler);
+      } else if (roll < 0.1) {
+        msg.channel.send({ files: [new Discord.AttachmentBuilder('media/buttermelonMan.jpeg')] }).catch(u.errorHandler);
+      } else {
+        msg.channel.send(`*buttermelon`).catch(u.errorHandler);
+      }
+    }
+  }
+}
+/**
+ * function slashFunQuote
+ * @param {Discord.ChatInputCommandInteraction} int a /fun quote interaction
+ */
+async function slashFunQuote(int) {
+  const url = `https://api.forismatic.com/api/1.0/?method=getQuote&format=json&lang=en`;
+  const response = await axios({ url, method: `get` }).catch((/** @type {axios.AxiosError} */ e) => {
+    throw new Error(`quote command error:${e.status}`);
+  });
+  const data = response.data;
+  if (data) {
+    const randomQuote = data;
+    return int.editReply(`> ${randomQuote.quoteText}\n> - ${randomQuote.quoteAuthor}`);
+  } else {
+    return int.editReply(`> A developer uses dark mode because bugs are attracted to light, \n` +
+    `> but wouldn't that put the bugs in the code instead of the background?\n` +
+    `> - ChainSword20000`);
+  }
+}
+/**
+ * function namegame
+ * @param {Discord.ChatInputCommandInteraction} int a /fun namegame interaction
+ */
+async function slashFunNamegame(int) {
+  let nameArg = int.options.getString(`name`);
+  try {
+    if (!nameArg) nameArg = int.user.displayName;
+    nameArg = nameArg.replace(/[^a-zA-Z]/g, '_');// just ABCabc etc, numbers were causing problems.
+    nameArg = nameArg.split(`_`)[0];// and just one segment
+    const name = nameArg;
+    const url = `https://thenamegame-generator.com/lyrics/${name}.html`;
+    const response = await axios({ url, method: `get` }).catch((/** @type {axios.AxiosError} */ e) => {
+      int.editReply(`Could not generate lyrics for ${name}.\nPerhaps you can get it yourself from https://thenamegame-generator.com.`);
+      throw new Error(`namegame command error:${e.status}`);
+    });
+    const data = response.data;
+    if (data) {
+      const profanityFilter = require(`profanity-matcher`);
+      const pf = new profanityFilter();
+      const lyricsUntrimmedEnd = data.substring(data.indexOf(`<blockquote>`) + 12);
+      const lyricsTrimmedWithHtml = lyricsUntrimmedEnd.substring(0, lyricsUntrimmedEnd.indexOf(`</blockquote>`));
+      const results = lyricsTrimmedWithHtml.replace(/<br>/g, `\n`).replace(/<br \/>/g, `\n`);
```

Comment:
You could use a regex exec for this. 
```js
/<blockquote>(.*)</blockquote>/gi
...
.replace(/<br ?/>/, "\n")
```

Reply:
I could, but hardly anyone can easily read regex, including me. indexof and substring is dead simple to understand and maintain in the future.